connectors/src/lib/structured_data.ts

```diff
@@ -1,21 +1,6 @@
 import { slugify } from "@dust-tt/types";
-import { hash as blake3 } from "blake3";
 
-// Extracting this helper due to incompatibility with blake3 in types.
-
-export function makeStructuredDataTableName(name: string, externalId: string) {
-  // Compute a blake3 hash of the externalId and name to avoid conflicts.
-  const hash = blake3(`${externalId}-${name}`).toString("hex");
-
-  // Define the prefix as the first 6 characters of the hash.
-  const externalIdPrefix = hash.substring(0, 6);
-
-  // Use the 6 last characters of the external id as a suffix (for user mapping to external data).
-  const externalIdSuffix = externalId.toLowerCase().slice(-6);
-
-  // Keep a maximum of 32 characters of the name.
-  const truncatedName = name.substring(0, 32);
-
-  // Concatenate everything.
-  return slugify(`${truncatedName}_${externalIdPrefix}_${externalIdSuffix}`);
+export function makeStructuredDataTableName(name: string) {
+  // Keep a maximum of 32 characters of the name and slugify it.
+  return slugify(name.substring(0, 32));
```

Comment:
Should we be smarter for google drive? Given that we define it as `${spreadsheet.title} - ${title}` if the spreadsheet title is longer than 32 we will always have the same name. I know that it does not cause conflicts in the DB anymore but will be a pain for the user and likely for the model as well?

We could do 16 of each or something along those lines?

Reply:
That is fair 👍 

Will then remove this function and make one gdrive specific and one notion specific.